unit-tests/src/main/scala/scala/scalanative/SyscallsSuite.scala

```diff
@@ -0,0 +1,63 @@
+package scala.scalanative
+
+import native._
+import posix.unistd
+import scala.scalanative.native.Nat._2
+
+object SyscallsSuite extends tests.Suite {
+
+  test("pipe + dup + getpid + getppid + fork + read + write") {
```

Comment:
Tests should be more finer-grain, please split into smaller ones that isolate minimum number of calls at a time.